3d_segmentation/unetr_segmentation_3d.ipynb

```diff
@@ -0,0 +1,4317 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            if img_name=='img0039.nii.gz':
</pre><p>If you only print 1 image, I think we don't need a for loop here?</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Fixed.